lib/engine/ability/README.md

```diff
@@ -107,7 +107,9 @@ Provide a description for an ability that is implemented outside of the ability
 
 Exchange this company for a share of a corporation.
 
-- `corporation`: The corporation whose share may be exchanged. Use `"any"` to allow for all corporations.
+- `corporation`: A single corporation, an array of corporations, or `"any"`.
```

Comment:
no, this is bad, change it to be corporations so it's not ambiguous what corporation is

Reply:
there can be corporation which is used when you expect one,

or corporations, when you expect multiple

Reply:
or you should change everything to use corporations

choose one approach

Reply:
choices

corporation and corporations attributes -- this seemse asier

only corporations attribute